rosidl_typesupport_cpp/QUALITY_DECLARATION.md

```diff
@@ -68,31 +69,37 @@ All pull requests must resolve related documentation changes before merging.
 
 ### Public API Documentation [3.ii]
 
-`rosidl_typesupport_cpp` has documentation of its public API, but it is not yet hosted.
+`rosidl_typesupport_cpp` has documentation of its public API, but it is not yet publicly hosted.
```

Comment:
This is not publicly available -> http://docs.ros2.org/latest/api/rosidl_typesupport_cpp/index.html

I will include it

Reply:
Sorry, I don't think I understand this comment.  As you say, the documentation isn't publically available, but this quality declaration is also saying "it is not yet publicly hosted".  So those would seem to agree.

Reply:
PR is open https://github.com/ros2/docs.ros2.org/pull/37 to make it public

Reply:
The documentation PR has now been merged. You can link to the api link @ahcorde gives above.